doc/api/stream.md

```diff
@@ -896,10 +896,9 @@ to consume data from a single stream. Specifically, using a combination
 of `on('data')`, `on('readable')`, `pipe()`, or async iterators could
 lead to unintuitive behavior.
 
-Use of the `readable.pipe()` method is recommended for most users as it has been
-implemented to provide the easiest way of consuming stream data. Developers that
-require more fine-grained control over the transfer and generation of data can
-use the [`EventEmitter`][] and `readable.on('readable')`/`readable.read()`
+`readable.pipe()` provides the easiest way to consume stream data. Developers
```

Comment:
This statement has not been true for _years_ (at least since we had async/iterators support).

I would remove these 3 lines altogether or maybe replace them with:

`readable.pipe` is a low-level API to connect streams. It is preferred to use [`pipeline`][] or [`compose`][] to connect streams. For convenience you can iterate streams with `for... await` loops or `.toArray()` a stream which is convenient but not as performant. 

Or a better phrasing of that